src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/processedenvelopes/EnvelopeProcessResult.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.domains.processedenvelopes;
+
+public class EnvelopeProcessResult {
+
+    public final Long ccdId;
+    public final EnvelopeCcdAction processedCcdType;
```

Comment:
`ccdActionType`? Not strong about it, just to apply or reject.

Reply:
ccdAction ?